app/layout.tsx

```diff
@@ -19,7 +20,8 @@ export default async function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body>
+      <body className={cn(
```

Comment:
is the `cn` util function required here? would just passing the className not be sufficient?

Reply:
I'll look deeper into this but, in the shad ui docs this seemed to be required. Once added I was able to see the old components.